docs/en/user-manual/command_parameters.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+lang: en
+nav_link: Command Parameters
+nav_level: 2
+nav_order: 40
+---
+
+
+# Command Parameters
+
+---
+
+[Previous Section](configuring_user_sync_tool.md)  \| [Next Section](usage_scenarios.md)
+
+---
+Once the configuration files are set up, you can run the User
+Sync tool on the command line or in a script. To run the tool,
+execute the following command in a command shell or from a
+script:
+
+`user-sync` \[ _optional parameters_ \]
+
+The tool accepts optional parameters that determine its
+specific behavior in various situations.
+
+
+| Parameters&nbsp;and&nbsp;argument&nbsp;specifications | Description |
```

Comment:
It would be great to replace this table with "dictionary list" style text.  Something like:
```
-h
--help
```
Show this help message and exit.

```
--users all
--users file <input_path>
--users group "<group1>,<group2>,..."
--users mapped
```
Specify the users to be selected for sync. The default is `all` meaning all users found in the directory. Specifying `file` means to take input user specifications from the CSV file named by the argument. Specifying `group` interprets the argument as a comma-separated list of groups in the enterprise directory, and only users in those groups are selected. Specifying `mapped` is the same as specifying `group` with all groups listed in the group mapping in the configuration file. This is a very common case where just the users in mapped groups are to be synced.

... and so on.